dev-python/berkeleydb/berkeleydb-18.1.15.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2025 Gentoo Authors
```

Comment:
You also need to fix the build backend upstream. Using legacy backend in `pyproject.toml` is plain invalid.

Reply:
Sedding it is just fine?

Reply:
Yes, but also send a proper patch upstream. They're working around missing `sys.path` adjustment.